packages/tocco-ui/src/Upload/Upload.js

```diff
@@ -6,7 +6,7 @@ import UploadProgress from './UploadProgress'
 import UploadInput from './UploadInput'
 
 /**
- * Description
+ * Component to upload files. If uploaded, the the gets displayed with Preview
```

Comment:
the the --> but I would adjust the sentence to :If uploaded, the file will be displayed by the Preview component